### PR TITLE
Fix #164: Replace Showtim3 with DubeySandeep

### LINKED ISSUE #164: Change tagged person for the 'good first issues' label
Instead of tagging Showtime3, Oppiabot should tag DubeySandeep when someone assigns the 'good first issue' tag https://github.com/oppia/oppia/issues/11313#issuecomment-737695429

### TODO

- [ ] Replace Showtime3 with DubeySandeep in the comment
- [ ] Find other mentions of Showtim3 in the oppiabot codebase and replace him with appropriate person
- [ ] When some person is included in a comment preferably allow configuring that person in https://github.com/oppia/oppiabot/blob/master/userWhitelist.json

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
Fixes #164: Replaces all mentions of Showtim3 with DubeySandeep.
Also, allow configuring users via [userWhitelist.json](https://github.com/oppia/oppiabot/blob/master/userWhitelist.json).

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: userWhitelist.json [hunk 1]
```diff
@@ -22,10 +22,12 @@
     "aks681"
   ],
   "teamLeads": {
-    "onboardingTeam": "Showtim3",
+    "onboardingTeam": "DubeySandeep",
     "releaseTeam": "ankita240796",
```
Comment (vojtechjelinek) [inline userWhitelist.json:26]: This is now me and Nithesh but I guess you can add just me for now.

File: userWhitelist.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "kevintab95",
     "seanlip",
     "ankita240796",
-    "Showtim3",
+    "DubeySandeep",
```
Comment (vojtechjelinek) [inline userWhitelist.json:16]: Maybe also add Srijan Redy to this list.

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thx, just a two nits.

File: userWhitelist.json (hunk 2 above)
Comment (jameesjohn) [reply userWhitelist.json:16]: Done.

File: userWhitelist.json (hunk 1 above)
Comment (jameesjohn) [reply userWhitelist.json:27]: Done.

Comment (jameesjohn) [reply userWhitelist.json:27]: Uhmm this means that oppiabot will ping you for good first issues since we are currently pinging the onboarding team lead.

Comment (vojtechjelinek) [reply userWhitelist.json:27]: I meant to replace Ankita with me, and keep DubeySandeep.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**